deploy/base/maven-v0.2.yaml

```diff
@@ -84,6 +84,8 @@ spec:
       description: The total number of dependencies that came from an untrusted source, such as Maven central
     - name: JAVA_COMMUNITY_DEPENDENCIES
       description: The list of community dependencies
+    - name: JAVA_DEPENDENCIES
```

Comment:
I am not sure about this approach, as it is an extra very long results field. An alternative would be to allow the analyser to create it directly, although that would require making sure the pipeline account had the correct permissions.

This was actually how things worked in the pre-kcp code.